treble_app/forms.py

```diff
@@ -10,7 +10,8 @@ class SongForm(forms.ModelForm):
     genre = forms.CharField(max_length=128)
     album = forms.CharField(max_length=128)
     no_of_recommendations = forms.IntegerField(widget=forms.HiddenInput(), initial=0)
-    recommended_songs = forms.ModelMultipleChoiceField(queryset=None, widget=forms.CheckboxSelectMultiple())
+    # TODO Verify that this is actually correct, was previously a merge conflict
```

Comment:
@2247469r Is the change in queryset here actually valid? I'm not 100% sure. (also applies a couple of lines down).

Reply:
I'm not sure about the exact way that `ModelMultipleChoiceField` works. I had a quick read through the docs, but it wasn't completely obvious. I'll have another look at it.